src/main/java/org/zalando/nakadi/controller/EventStreamController.java

```diff
@@ -84,6 +88,7 @@ public EventStreamController(final EventTypeRepository eventTypeRepository, fina
         this.closedConnectionsCrutch = closedConnectionsCrutch;
         this.blacklistService = blacklistService;
         this.consumerLimitingService = consumerLimitingService;
+        this.featureToggleService = featureToggleService;
```

Comment:
As I understand the idea was that we will be able to switch it off if it puts too high load on Zookeeper.